test/integration/dockerregistryclient_test.go

```diff
@@ -97,11 +99,14 @@ func TestRegistryClientConnectPulpRegistry(t *testing.T) {
 	}
 
 	var image *dockerregistry.Image
-	err = retryWhenUnreachable(t, func() error {
+	err = retryWhenUnreachableOrNotFound(t, func() error {
 		image, err = conn.ImageByTag("library", "rhel", "latest")
 		return err
 	})
 	if err != nil {
+		if strings.Contains(err.Error(), "x509: certificate has expired or is not yet valid") {
+			t.Skip(err)
```

Comment:
I know, integration tests will log this as:

    ok      TestRegistryClientConnectPulpRegistry

this is just following another precedence on line [228](https://github.com/openshift/origin/pull/6335/files#diff-67b0e91215f1c0300b678ffbb343b89dR228).